presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClusteringPredicatesExtractor.java

```diff
@@ -25,7 +25,6 @@
 import io.prestosql.spi.predicate.TupleDomain;
```

Comment:
Update commit message to `Add TuppleDomain#filter`

Reply:
I chose `Add TupleDomain transformations` because I add two methods: `filter` and `transformDomains`.